src/ports-overlay/librpm/portfile.cmake

```diff
@@ -6,6 +6,11 @@ vcpkg_from_github(
   HEAD_REF rpm-4.18.2-release
 )
 
+vcpkg_replace_string("${SOURCE_PATH}/configure.ac"
+  "RPMCONFIGDIR=\"`echo \${usrprefix}/lib/rpm`\""
+  "RPMCONFIGDIR=\"/usr/lib/rpm\""
+)
```

Comment:
This looks great! but just out of curiosity, have you tried using the `--prefix` in the configure below? I'm just looking at the configure file from librpm and it goes something like this:

```
if test X"$prefix" = XNONE ; then
    usrprefix="$ac_default_prefix"
else
    usrprefix=$prefix
fi

RPMCONFIGDIR="`echo ${usrprefix}/lib/rpm`"
```

I'm happy to go with this option too, but I'm curious about it.

Reply:
Yes, I tried it. The problem is that _vcpkg_ also adds the `--prefix` and replaces the one set with the `vcpkg_configure_make()` function. Here is the evidence: https://github.com/wazuh/wazuh-agent/issues/404#issuecomment-2537415773

Reply:
Got it, thanks !